frog/imports/client/SingleActivity/TopBar.js

```diff
@@ -1,29 +1,39 @@
 // @flow
 
-import React from 'react';
+import * as React from 'react';
 import { withStyles } from '@material-ui/styles';
 import AppBar from '@material-ui/core/AppBar';
 import Toolbar from '@material-ui/core/Toolbar';
 import Typography from '@material-ui/core/Typography';
 import Button from '@material-ui/core/Button';
 import { type PropsT } from './types';
 import { style } from './style';
+import { withModalController } from '../Wiki/components/Modal';
```

Comment:
Move the `Modal` to `frog-utils`. We want to avoid cross-dependencies across the various features of FROG.

Reply:
Maybe we should consider doing this in a separate PR when we look at how to make it more flexible with showing and hiding etc.. Would love your thoughts